plat/ti/k3/common/plat_topology.c

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2017-2018, ARM Limited and Contributors. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <platform_def.h>
+#include <psci.h>
+
+/* The power domain tree descriptor */
+static unsigned char power_domain_tree_desc[] = {
+	PLATFORM_CLUSTER_COUNT,
+	ARM_CLUSTER0_CORE_COUNT,
+#if ARM_CLUSTER1_MSMC_PORT != UNUSED
+	ARM_CLUSTER1_CORE_COUNT,
+#endif /* ARM_CLUSTER1_MSMC_PORT != UNUSED */
+#if ARM_CLUSTER2_MSMC_PORT != UNUSED
```

Comment:
Again comment on names for macros. Would prefer `K3_` as prefix for these macros.

Reply:
Will fix.